hbw/app/javascript/packs/hbw/components/form/string.js.jsx

```diff
@@ -64,10 +64,15 @@ modulejs.define('HBWFormString', ['React'], (React) => {
       return <div className={rootCSS} title={params.tooltip}>
         <div className="form-group">
           <span className={labelCss}>{label}</span>
+          {params.description && params.description.placement === 'top'
```

Comment:
You can use safe-navigation operator like `params.description?.placement === 'top'`